README.md

```diff
@@ -72,24 +72,38 @@ of the Azure Active Directory window respectively as *Name* and *Directory ID*
 
 #### Register the service app (TodoListService-NativeDotNet)
 
-1. In the  **Azure Active Directory** pane, click on **App registrations** and choose **New application registration**.
-1. Enter a friendly name for the application, for example 'TodoListService-NativeDotNet' and select 'Web app / API' as the *Application Type*.
-1. For the *Sign-on URL*, enter the base URL for the sample. By default, this sample uses `https://localhost:44321/`.
-1. Click **Create** to create the application.
-1. In the succeeding page, Find the *Application ID* value and record it for later. You'll need it to configure the Visual Studio configuration file for this project.
-1. Then click on **Settings**, and choose **Properties**.
-1. For the App ID URI, replace the guid in the generated URI 'https://\<your_tenant_name\>/\<guid\>', with the name of your service, for example, 'https://\<your_tenant_name\>/TodoListService-NativeDotNet' (replacing `<your_tenant_name>` with the name of your Azure AD tenant)
+1. In the  **Azure Active Directory** pane, click on **[App registrations](https://go.microsoft.com/fwlink/?linkid=2083908)** and choose **New registration**.
+1. Enter a friendly name for the application, for example `TodoListService-NativeDotNet`.
+1. Leave **Supported account types** on the default setting of **Accounts in this organizational directory only**.
+1. For the *Redirect URI (optional)*, select "Web" from the dropdown, and enter the base URL for the sample. By default, this sample uses `https://localhost:44321/`.
+1. Click **Register** to create the application.
+1. In the succeeding page, Find the *Application (client) ID* value and record it for later. You'll need it to configure the Visual Studio configuration file for this project.
+1. In the list of pages for the app, select on **Expose an API**
+   - Select **Add a scope**
+   - accept the proposed Application ID URI (api://{clientId}) by selecting **Save and Continue**
+   - Enter the following parameters
+     - for **Scope name** use `access_as_user`

```

Comment:
[Information]
v1.0 applications used to add automatically the  `user_impersonation` scope. But given that the v1.0 libraries don't use scopes but only resources, I think this is ok